Session.js

```diff
@@ -1391,15 +1405,35 @@ Session.prototype = {
 			return getDocumentActiveElement();
 		}
 		else {
-			return this._post('element/active').then(function (element) {
+			var activeFunc = self.capabilities.useGetForActiveElement ?
+				self._get :
+				self._post;
+			return activeFunc.call(this, 'element/active').then(function (element) {
```

Comment:
The mixing of `self` and `this` in the same scope is kind of confusing.

Reply:
Fixed